Lab 2.ipynb

```diff
@@ -0,0 +1,1490 @@
+      "text/plain": [
+       "<Figure size 576x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "X = 3.7\n",
+    "Y = 7.2\n",
+    "\n",
+    "# X = average cosmic-ray background\n",
+    "# Y = average number of gamma-rays emitted by hypothetical source\n",
+    "\n",
+    "# Part A ---\n",
+    "\n",
+    "k = np.arange(0,20,1)\n",
+    "day_1 = stats.poisson.pmf(k, X, loc = 0)\n",
+    "\n",
+    "fig, ax = plt.subplots(1, 1, figsize=(8,8))\n",
+    "\n",
+    "plt.tick_params(labelsize = 12)\n",
+    "ax.set_xlim([0,10])\n",
+    "ax.set_ylabel('Probability', fontsize = 12)\n",
+    "ax.set_xlabel('Number of Gamma-rays')\n",
+    "ax.set_title(\"Gamma-ray Occurrences\")\n",
+    "\n",
+    "ax.step(k, day_1, 'o-',where = 'mid')\n",
+    "\n",
+    "day_2 = np.convolve(day_1, day_1)\n",
+    "k2 = np.arange(0, len(day_2), 1)\n",
+    "ax.step(k2 / 2, day_2, 'o-', where = 'mid')\n",
+    "\n",
+    "day_3 = np.convolve(day_1, day_2)\n",
+    "k3 = np.arange(0, len(day_3), 1)\n",
+    "ax.step(k3  / 3, day_3, 'o-', where = 'mid')\n",
+    "\n",
+    "day_4 = np.convolve(day_1, day_3)\n",
+    "k4 = np.arange(0, len(day_4), 1)\n",
+    "ax.step(k4  / 4, day_4, 'o-', where = 'mid')\n",
+    "        \n",
+    "day_5 = np.convolve(day_1, day_4)\n",
+    "k5 = np.arange(0, len(day_5), 1)\n",
+    "ax.step(k5  / 5, day_5, 'o-', where = 'mid')\n",
```

Comment:
This plot could use an explanation, why does the mean stay the same // does it approach Gaussian? Maybe do another log plot